extension/test/unit/submit-payment-details.handler.spec.js

```diff
@@ -0,0 +1,144 @@
+    }
+  }
+  /* eslint-enable max-len */
+
+  beforeEach(() => {
+    scope = nock(`${config.adyen.apiBaseUrl}`)
+  })
+
+  afterEach(() => {
+    nock.cleanAll()
+  })
+
+  it('when resultCode from Adyen is "Authorized", '
+    + 'then it should return actions "addTransaction", "addInterfaceInteraction" and "setCustomField"', async () => {
+    scope.post('/payments/details')
+      .reply(200, submitPaymentDetailsSuccessResponse)
+
+    const ctpPaymentClone = _.cloneDeep(ctpPayment)
+    ctpPaymentClone.custom.fields.submitAdditionalPaymentDetailsRequest = JSON.stringify(submitPaymentDetailsRequest)
+    ctpPaymentClone.custom.fields.makePaymentResponse = JSON.stringify(makePaymentRedirectResponse)
+
+    const response = await execute(ctpPaymentClone)
+    console.log(`response ${JSON.stringify(response.actions)}`)
+
+    expect(response.actions).to.have.lengthOf(3)
+    const addInterfaceInteraction = response.actions.find(a => a.action === 'addInterfaceInteraction')
+    expect(addInterfaceInteraction.fields.type).to.equal('submitPaymentDetails')
+    expect(addInterfaceInteraction.fields.request).to.be.a('string')
```

Comment:
assertions in here do not give me an idea, it is ok for createdAt but not easy to understand for request, what we are expecting here is not asserted, same for all similarly checks on this method.

Reply:
Check this: https://github.com/commercetools/commercetools-adyen-integration/pull/271/files/86d1d1156eac7bc65958344bda53bb36694e60e2#diff-2f7b0026dab2a37eb3a856341371ecdaR51-R57